### DISCUSSION TITLE
Hello world on https://www.goparamore.io/hello-world implementing wrong interface

### DISCUSSION THREAD

**Discussion by sukkergris** [BrighterCommand/Brighter#2132] [General]
Hi There. Just love this lib! I'm diving in the documentation these days and just found this minor error. In the docs the interface IAmAHandlerFactory is used. It should be IAmAHandlerFactorySync. :)
I couldn't figure out where to fix it - hence this 'discussion'.

**Comment by holytshirt** [comment]
Hi @sukkergris  thank you so much. That bit of documentation needs a little love, really need to bring it up to date with the dotnet core samples we have in the repo

**Reply by sukkergris** [reply]
Can I help? :)

**Comment by holytshirt** [comment]
Let me check later I'm just traveling home currently. I think that code is
on a normal website which you can't PR.

We have more detailed documentation here
https://brightercommand.gitbook.io/paramore-brighter-documentation/
Which you can PR 😁
